library/src/pivotal-ui/components/forms/forms.scss

```diff
@@ -285,9 +285,48 @@ This form looks and behaves mostly like a content editable field
     cursor: pointer;
     outline: none;
     text-indent: -99999px;
-    width: $toggle-bg-width;
-    height: $toggle-bg-height;
-    border-radius: $toggle-bg-height;
+
```

Comment:
@charleshansen 👍 a19c22dab44ee86a57ffb03dec7814b112358ac8 thoughts on naming?

Also, what was the `bg` acronym?